tests/MercadoPago/Client/Base/BaseClient.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace MercadoPago\Client\Base;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Base Client.
+ */
+abstract class BaseClient extends TestCase
+{
+    protected function mockHttpRequest(string $filepath, int $status_code): \PHPUnit\Framework\MockObject\MockObject|\MercadoPago\Net\HttpRequest
+    {
+        /** @var \PHPUnit\Framework\MockObject\MockObject|\MercadoPago\Net\HttpRequest $mockHttpRequest */
+        $mockHttpRequest = $this->getMockBuilder(\MercadoPago\Net\HttpRequest::class)->getMock();
```

Comment:
Seguir com padrão snake_case
```suggestion
        $mock_http_request = $this->getMockBuilder(\MercadoPago\Net\HttpRequest::class)->getMock();
```

Reply:
Ajustado!